README.md

```diff
@@ -50,4 +50,11 @@ python GraveIdentifyAndRemove.py [h5m file] -o [output file]
 
 ----------------------------------------
 
+3. TagExpansion.py can be used to expand vector tags on a mesh and generate a data file for each time state. VisIt will recognize the collection of these data files as a database, and permit stepping through them in an animation.
```

Comment:
Same comment on time state vs. index as above. Maybe something like "for each value in the vector tag"?